tsl/src/remote/dist_ddl.c

```diff
@@ -681,6 +681,8 @@ dist_ddl_process_alter_table(ProcessUtilityArgs *args)
 			case AT_AddConstraintRecurse:
 			case AT_DropConstraint:
 			case AT_DropConstraintRecurse:
+			case AT_SetNotNull:
+			case AT_DropNotNull:
```

Comment:
Why does adding these lines solve the problem?

Reply:
That's basically a whitelist of ALTER commands that can be run on dist hypertable